kubernetes/rucio/README.md

```diff
@@ -0,0 +1,146 @@
+Adapted from rucio instructions here: https://github.com/rucio/helm-charts/tree/master/rucio-server
+
+
+# First time setup
+
+## Setup a new cluster in the CMSRucio project:
+
+
+    openstack coe cluster delete --os-project-name CMSRucio  cmsruciotest
+    openstack coe cluster create cmsruciotest --keypair lxplus  --os-project-name CMSRucio   --cluster-template kubernetes-preview --node-count 4
+    openstack coe cluster list --os-project-name CMSRucio # Monitor creation status
+
+
+### If setting up a new/changed cluster:
+
+
+    rm key.pem cert.pem ca.pem config
+    openstack coe cluster config  --os-project-name CMSRucio  cmsruciotest
+    export KUBECONFIG=/afs/cern.ch/user/e/ewv/config
+
+
+Copy and paste the last line. On subsequent logins it is all that is needed. Now make sure that your nodes exist:
+
+    -bash-4.2$ export KUBECONFIG=/afs/cern.ch/user/e/ewv/config
+    -bash-4.2$ kubectl get nodes
+    NAME                                 STATUS    ROLES     AGE       VERSION
+    cmsruciotest-mzvha4weztri-minion-0   Ready     <none>    5m        v1.11.2
+    cmsruciotest-mzvha4weztri-minion-1   Ready     <none>    6m        v1.11.2
+    cmsruciotest-mzvha4weztri-minion-2   Ready     <none>    6m        v1.11.2
+    cmsruciotest-mzvha4weztri-minion-3   Ready     <none>    6m        v1.11.2
+
+## Install helm if needed
+
+    mkdir $HOME/bin
+    export PATH=$PATH:$HOME/bin
+    cp helm $HOME/bin
```

Comment:
It is not clear where user should get helm in a first place. Please provide instructions how to install helm, e.g. clone github repo, build it, or install through rpms, etc.

Reply:
You'll be happy to know that it's a Go binary you just download. :-)